pages/api/github/callbacks/webhook.ts

```diff
@@ -43,30 +43,32 @@ const webhookHandler = async (req: NextApiRequest, res: NextApiResponse) => {
 		res.status(400).json({ error: 'Unable to get repoConfig from db' });
 		return;
 	}
-	const data = {
-		repositoryProvider: 'github',
-		eventPayload: jsonBody,
-		repoConfig: repoConfig,
-		eventType: event_type
-	};
-	
-	const msgType = 'webhook_callback';
-	const topicName_str = topicName;
-	
-	console.debug(`[webookHandler] data = ${JSON.stringify(jsonBody)}`)
-	console.debug(`[webookHandler] topicname = ${topicName}`)
-	console.debug(`[webookHandler] repoConfig = ${JSON.stringify(repoConfig)}`)
-	
-	const result: string | null = await publishMessage(topicName_str, data, msgType)
-	.catch((error) => {
-		console.error('[webookHandler] Failed to publish message:', error);
-		return null;
-	});
-	if (result == null) {
-		res.status(500).json({ error: 'Internal Server Error' });
-		return;
+	// Publish message to Pub/Sub for each install_id (topic name)
+	for (const installId of topicName) {
+		const data = {
+			repositoryProvider: 'github',
+			eventPayload: jsonBody,
+			repoConfig: repoConfig,
+			eventType: event_type
+		};
+
+		const msgType = 'webhook_callback';
+
+		console.debug(`[webookHandler] data = ${JSON.stringify(jsonBody)}`);
+		console.debug(`[webookHandler] installId = ${installId}`);
+		console.debug(`[webookHandler] repoConfig = ${JSON.stringify(repoConfig)}`);
+
+		const result: string | null = await publishMessage(installId, data, msgType)
+		.catch((error) => {
+			console.error('[webookHandler] Failed to publish message:', error);
+			return null;
+		});
+		if (result == null) {
+			res.status(500).json({ error: 'Internal Server Error' });
```

Comment:
continue if error comes. count number of failures. Send 200 only if there is no failure